.flake8

```diff
@@ -10,6 +10,10 @@ ignore =
     # high Jones complexity isn't always so bad
     WPS221,
 
+    # more than 3 `elif` branches will not be a problem
+    # the solutions found in network (like creating a class to do the `elif`) will make the code complicated
+    WPS223,
```

Comment:
I wonder if `match` and `case` would be more clear. They added those in Python 3.10, apparently

Reply:
fixed